CONTRIBUTING.md

```diff
@@ -497,6 +497,88 @@ rebase the branch on main, fixing any conflicts along the way before the code ca
 
 ---
 
+## Adding a new ACL Token
+
+Checklist for getting server-acl-init to generate a new ACL token assuming the new token
+is named `foo`.
+
+### Control Plane
+
+* `control-plane/subcommand/server-acl-init/command.go`
+    * Add `flagCreateFooToken bool` to vars list
+    * Initialize flag in `init`
```

Comment:
```suggestion
    * [ ] Add `flagCreateFooToken bool` to vars list
    * [ ] Initialize flag in `init`
```

Reply:
I would do this throughout for the sub-items because it will make a nice checklist the user can click to tick off items.

Reply:
I thought about this but how can they check off the items in the contributing.md though?

Reply:
I think it just sets a toggle in `localStorage`. Does it work on your branch if you add it?

Reply:
nope 😢 : https://github.com/hashicorp/consul-k8s/blob/ce5916890b8a280e86c9914fae54b9b743c20bc0/CONTRIBUTING.md